README.md

```diff
@@ -62,16 +65,21 @@ cmake -DCMAKE_CURRENT_BINARY_DIR=$PWD \
 
 ### Install
 
+Once cmake configuration successfully completed, from the same build directory run build, install commands
+
 ```shell
-         make
-         make install
+cmake --build .
+cmake --install .
```

Comment:
This should be 
cmake --build . --
cmake --build . -- install

the syntax is "cmake" "--build" the directory to build in, "--" (end of cmake args) then any args to give to the build program.

Reply:
Thanks Updated the commands in the latest version.